packages/node/rollup.config.js

```diff
@@ -23,15 +22,31 @@ export default {
             file: packageJson.module,
             format: "esm",
             sourcemap: true,
-        },
+        }
     ],
+    resolveConfig: {
+        extensions,
+    },
+}, {
+    output: {
+        exports: "named",
+        format: "es",
+        file: "dist/browser/index.js",
+        sourcemap: true,
+    },
+    resolveConfig: {
+        extensions,
+        exportConditions: ["browser", "worker"],
+        browser: true,
+    },
+}].map(config => ({
```

Comment:
The detour via `array.map` was necessary since `@rollup/plugin-node-resolve` can to be used as a rollup output plugin.